### DISCUSSION TITLE
V121 Huge memory leak

### DISCUSSION THREAD

**Discussion by oculunto** [seleniumbase/SeleniumBase#2461] [Q&A] [answered]
Hello, 
 Since chrome V121 I'm having a huge memory leak, memory usage keeps increasing after every refresh. I am trying to use another chrome version by using  `browser = Driver(uc=True,driver_version=120")` but it keeps using V121 (I figured it changes only the driver but not the current chrome version). Is there a way to also specify which chrome version to use or does it automatically use the default installed one?

**Comment by oculunto** [comment, marked as answer]
I found a solution: 
I installed chrome beta V122 and used `binary_location=path` to specify the path to the chrome executable.

**Reply by vyanezz** [reply]
I also get the "Out of memory" error calling driver.get(). 

Can you send this chrome beta download link? Only need to change the ` binary_location=path` param of the driver? 

Thanks.

**Reply by mdmintz** [reply]
It's the same thing seen in https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1735

All Chrome v121 users will be hitting it until patched, but this issue is probably more easily seen with automation frameworks because Chrome is starting and quitting a lot more often.

You can get Chrome Beta releases here: https://www.google.com/chrome/beta/

Or if you prefer to keep Chrome v121 for now and you run A LOT of automated scripts, then maybe use a shell command that terminates all Chrome processes after every thousand tests, or something like that.
